sdk/recoveryservicesbackup/azure-resourcemanager-recoveryservicesbackup/src/test/java/com/azure/resourcemanager/recoveryservicesbackup/RecoveryServicesBackupManagerTests.java

```diff
@@ -0,0 +1,224 @@
+    private static final Random RANDOM = new Random();
+    private static final Region REGION = Region.US_WEST2;
+    private String resourceGroupName = "rg" + randomPadding();
+    private RecoveryServicesBackupManager recoveryServicesBackupManager;
+    private RecoveryServicesManager recoveryServicesManager;
+    private ResourceManager resourceManager;
+    private boolean testEnv;
+
+    @Override
+    public void beforeTest() {
+        final TokenCredential credential = new DefaultAzureCredentialBuilder().build();
+        final AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
+
+        recoveryServicesManager = RecoveryServicesManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile);
+
+        recoveryServicesBackupManager = RecoveryServicesBackupManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile);
+
+        resourceManager = ResourceManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile)
+            .withDefaultSubscription();
+
+        // use AZURE_RESOURCE_GROUP_NAME if run in LIVE CI
+        String testResourceGroup = Configuration.getGlobalConfiguration().get("AZURE_RESOURCE_GROUP_NAME");
+        testEnv = !CoreUtils.isNullOrEmpty(testResourceGroup);
+        if (testEnv) {
+            resourceGroupName = testResourceGroup;
+        } else {
+            resourceManager.resourceGroups()
+                .define(resourceGroupName)
+                .withRegion(REGION)
+                .create();
+        }
+    }
+
+    @Override
+    protected void afterTest() {
+        if (!testEnv) {
+            resourceManager.resourceGroups().beginDeleteByName(resourceGroupName);
+        }
+    }
+
+    @Test
+    @DoNotRecord(skipInPlayback = true)
+    public void testCreateProtectionPolicy() {
+        Vault vault = null;
+        ProtectionPolicyResource protectionPolicyResource = null;
+        String randomPadding = randomPadding();
+        try {
+            String vaultName = "vault" + randomPadding;
+            String policyName = "policy" + randomPadding;
+
+            // @embedmeStart
+            List<SubProtectionPolicy> lstSubProtectionPolicy = Arrays.asList(
+                new SubProtectionPolicy()
+                    .withPolicyType(PolicyType.FULL)
+                    .withSchedulePolicy(
+                        new SimpleSchedulePolicy()
+                            .withScheduleRunFrequency(ScheduleRunType.WEEKLY)
+                            .withScheduleRunDays(Arrays.asList(DayOfWeek.SUNDAY, DayOfWeek.TUESDAY))
+                            .withScheduleRunTimes(Arrays.asList(OffsetDateTime.parse("2023-06-13T10:00:00Z"))))
```

Comment:
Maybe use `OffsetDateTime.now().plus(1, ChronoUnit.DAYS)` to avoid scheduling in past time? Same for other OffsetDateTimes.

Reply:
I've tried this and found that only the following date formats work with `yyyy-mm-ddThh:mm:ssZ`

Reply:
Got it. You could try 
```java
OffsetDateTime.parse(OffsetDateTime.now().plusDays(1).format(DateTimeFormatter.ofPattern("yyyy-MM-dd'T'hh:mm:ss'Z'")))
```

Though it's a bit overkill..